packages/core/package.json

```diff
@@ -65,7 +65,7 @@
     "!src/main/**/*test*.ts"
   ],
   "license": "GPL-3.0",
-  "main": "src/main/Account",
+  "main": "src/main/index",
```

Comment:
This will help us move from outputting the built file in `src` folder. We can here export whatever is needed to the consumer